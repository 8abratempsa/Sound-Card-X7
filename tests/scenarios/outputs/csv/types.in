octosql "SELECT 42,
                42.42, 42.42424242424242, float(42),
                true, false,
                null,
                'test' as hello,
                INTERVAL 5 HOURS + INTERVAL 32 MINUTES + INTERVAL 42 SECONDS" -ocsv
