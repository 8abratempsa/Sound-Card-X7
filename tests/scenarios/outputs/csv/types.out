col_0,col_1,col_2,col_3,col_4,col_5,col_6,hello,col_8
42,42.42,42.42424242424242,42,true,false,,test,5h32m42s
